IFC4x3/Properties/s/StationsViewAllStaffPositions_2$H1tVSRb5I9MS8oseNlm7/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="StationsViewAllStaffPositions_2H1tVSRb5I9MS8oseNlm7" Name="StationsViewAllStaffPositions" UniqueId="bf441ddf-71b9-4548-959c-232da85efc07" PrimaryDataType="IfcText" />
```

Comment:
Looking at the description, would it make sense to rename this?
e.g. "HinderingObstactlesDescription"